res/database/Default/user/scr0llbaer.xml

```diff
@@ -0,0 +1,1586 @@
+			<title>
+				<de>%title%</de>
+				<en>%title%</en>
+			</title>
+			<description>
+				<de>Programmfüller für den jeden Morgen. Hier werden nochmal die Beiträge aus dem Mülleimer gefischt, die die Kollegen aus der Nachrichtenredaktion schon weggeschmissen haben. Andere niedrigschwellige Inhalte finden sich bestimmt auch noch.</de>
+				<en>Programme filler for every morning. Here the stories that the colleagues from the newsroom have already thrown away are fished out of the garbage can again. Other low-threshold content can surely be sourced as well.</en>
+			</description>
+
+			<variables>
+				<title>
+					<de>Morgenmuffel-TV|Mundgeruch am Morgen|Miesepeter am Morgen|Morgengrieß-Kram|G'n Morgääähn</de>
+					<en>Morning Grouch TV|Bad breath in the Morning|Grumpy in the Morning|Morning Flour and Stuff|G'd Moryaaawn</en>
+				</title>
+			</variables>
+
+			<jobs>
+				<job index="0" function="8" required="1" />
+			</jobs>
+			<data flags="1" broadcast_time_slot_start="5" broadcast_time_slot_end="10" production_limit="420" production_broadcast_limit="1" production_broadcast_flags="512" production_licence_flags="36" />
+			<genres maingenre="103" subgenres="6,300" />
+			<blocks value="3" />
+			<price min="2000" max="4000" />
+			<potential min="20" max="40" />
+			<outcome min="20" max="40" />
+			<review min="10" max="20" />
+			<speed min="10" max="20" />
+			<production_time value="60" />
+		</scripttemplate>
+
+		<scripttemplate product="3" licence_type="1" guid="d35ad24a-9095-490e-b5d1-f51ba82210ff" creator="9551" comment="Deluxe- und Endlos-Variante von scripttemplate-ron-morningshow">
+			<title>
+				<de>%title%</de>
+				<en>%title%</en>
+			</title>
+			<description>
+				<de>Es lohnt sich! Für jeden Morgen! Tolles Programm und gute Unterhaltung für einen beschwingten Start in den Tag.</de>
+				<en>It's worth it! For every morning! Great program and good entertainment for an upbeat start into the day.</en>
+			</description>
+
+			<variables>
+				<title>
+					<de>Das saftige Frühstücks-TV|5-Sterne Frühstück Deluxe|Morgenpracht-TV|Croissant mit Latte am Morgen TV</de>
+					<en>The Lush Breakfast TV|5-Star Breakfast Deluxe|Morning Glory TV|Croissant with Latte in the Morning TV</en>
+				</title>
+			</variables>
+
+			<jobs>
+				<job index="0" function="1" required="1" />
+				<job index="1" function="4" required="1" />
+				<job index="2" function="8" required="1" />
+				<job index="3" function="128" required="1" />
+				<job index="4" function="2048" gender="2" required="1" />
+				<job index="5" function="64" required="0" />
+				<job index="6" function="8" required="0" />
+				<job index="7" function="16" required="0" />
+			</jobs>
+			<data flags="1" broadcast_time_slot_start="5" broadcast_time_slot_end="11" production_limit="999" production_broadcast_limit="1" production_broadcast_flags="512" production_licence_flags="36" />
```

Comment:
An idea would be the possibility to eg have the studio manager ("drop it on him") capable of extending scripts (...raising costs with each extension). Could also be the script vendor ...

Reply:
I tried and it's already quite "fine" during gameplay.
I think it's "realistic" because also in the "real world", once you have a "concept", it shouldn't be necessary to "buy" a new one (that is essentially just the same with another name).
And what I also did with those "endless" scripts is the requirement to book a screenwriter (bookable screenwriters were previously unused, as far as I can tell). So from the script agency you "buy" the "concept", and the screenwriters write each episode.

Reply:
ok, changing to 100 for now, but if possible (as soon it's clear there are no undesirable side-effects) I'd eventually want to change it back, as sometimes the number is a cultural reference and part of the joke (like 1337).

though of course this may become moot once the format of such "endless" scripts is "officially" supported by the game.